causalml/match.py

```diff
@@ -195,7 +195,13 @@ def match(self, data, treatment_col, score_cols):
                     t_idx_matched.append(t_idx)
                     c_idx_matched.append(c_idx_min)
                     control.loc[c_idx_min, "unmatched"] = False
+        # Add the flags of the treatment and control groups to the matched dataframe
+        print(
+            "You can get matched dataframe where 't' represents the treatment group and 'c' represents the control group"
+        )
 
+        data.loc[np.array(t_idx_matched), "group"] = "t"
+        data.loc[np.array(c_idx_matched), "group"] = "c"
```

Comment:
This is redundant as it is already available in `treatment_col`.

Reply:
Hi @chenzhongd, I will close this PR for now. If you think my comment above is incorrect, please leave your thought and reopen the PR.